zebra-state/src/response.rs

```diff
@@ -137,6 +141,10 @@ pub enum ReadResponse {
     #[cfg(feature = "getblocktemplate-rpcs")]
     /// Response to [`ReadRequest::SolutionRate`](crate::ReadRequest::SolutionRate)
     SolutionRate(Option<u128>),
+
+    #[cfg(feature = "getblocktemplate-rpcs")]
+    /// Response to [`ReadRequest::CheckBlockValidity`](crate::ReadRequest::CheckBlockValidity)
+    ValidBlock(block::Hash),
```

Comment:
Same here, it would be good to be clear that these blocks aren't valid.

Reply:
Renamed in [`8420b75`](https://github.com/ZcashFoundation/zebra/pull/5870/commits/8420b75013c1a05ea11a9799d1f87bd338c3c27c).